### ISSUE TITLE
Support for RDF Star and SPARQL Star?

### ISSUE THREAD

**Issue by jpmccu** [RDFLib/rdflib#1406]
This may already have an issue, but is there a chance we can get support for RDF* and SPARQL* in RDFlib? It's becoming much more common with the expansion of Wikidata, and it might be nice to support its use.

**Comment by mwatts15** [comment]
Especially at this point, where SPARQL* is still in a draft phase, I think it would be preferable to define another package, like a "rdflib-star". It should be pretty feasible to implement SPARQL* support with a      rdflib.query.Processor plugin, possibly registered with a "rdf.plugins.queryprocessor" entry point. I'm thinking something like:
```
register("sparql*", Processor, "rdflib_star.plugins.sparql_star.processor", "SPARQLStarProcessor")
```

A copy of the existing SPARQL parser could be made more extendible though. As it stands, after you copy the root parser elements, you'll need to navigate through the grammar/graph of ParserElements to reach the elements you want to change. I'm thinking of a traversal function that allows you to either specify a name as it's declared in the parser module or specify a path and a replacement ParserElement.

**Comment by mwatts15** [comment]
This may be closed as a duplicate of  #955

**Comment by gjhiggins** [comment]
> This may be closed as a duplicate of #955

Closed as a duplicate of #955
